source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp

```diff
@@ -118,6 +130,12 @@ namespace
         printHeaderLine("Connecting and disconnecting ", numConnects, " times each (be patient):");
     }
 
+    void printCapture3DAndFiltersHeader(const std::string &subtest)
```

Comment:
From the arguments it looks like this function should be called `printSubtest`, or something like that.

Reply:
Done - printSubtestHeader